### PR TITLE
Remove surrounding navBarTitle div and Fix text decoration of NavBar title

### PR DESCRIPTION
**Pull request by GoodGameRuler**
# Remove surrounding navBarTitle div and simplify into just the link component

TLDR: You've added a wrapper div, I just removed it and fixed the text underline issue.

**First Note - try not to touch `reset.css` for reason unrelated to _reseting_. `reset.css` just removes any underlying css that browsers provide.** If you would like to remove underline (font text decoration then just create a new class that has that css).

Remember a general rule.
**Overrides**
- ID CSS
- Class CSS
- A Second Class CSS Applied to same element
- Element CSS 

**Gets Overridden**  

Also only add a surrounding div if 
- The component is complex. In this context imagine the navbar title contained some emojis and an image it would be more complex to manage all the components that the title has.
- There is some management of padding or margins you need specific to having a wrapper.
- A few other reasons I can't think of rn ;-;

Otherwise having a wrapper just complicates code, and hurts readability.

Another general rule is that all CSS is passed down to child divs so that usually means that we can move a lot of the work up or down, whichever makes life easier.

**Before**
image

**After**
image

### CODE REVIEW

**Review by jennifermtan — APPROVED** [review]
Looks good to me - thanks Udit!
